### ISSUE TITLE
Inconsistent rows result on sort when property template is used 

### ISSUE THREAD

**Issue by partikule** [json-editor/json-editor#789]
### General information

- json-editor version: 2.3.0

### Expected behavior

When sorting an array of items containing one template generated ID, the row stays consistent

id_1 | the_value_one
id_2 | the_value_two

Click on "move up" on the_value_two

The result is :

id_2 | the_value_two
id_1 | the_value_one

### Actual behavior

When sorting rows of items containing template generated value, each value stays in the origin rows : 

The result is :

id_1 | the_value_two
id_2 | the_value_one

### Steps to reproduce the behavior

Create the  initUniqueId() callback : 

```javascript
            JSONEditor.defaults.callbacks.template = {
                "initUniqueId": (jseditor,e) => {
                    if (jseditor.getValue())
                        return jseditor.getValue()
                    return (new Date().getTime()).toString(36)
                }
            }
```

Use the schema :

```javascript
{
  "$schema": "http://json-schema.org/draft-07/schema#",
  "title": "The schema",
  "type": "array",
  "format": "table",

  "definitions": {

    "item": {
      "type": "object",
      "properties": {
        "id_item": {
          "type": "string",
          "template": "initUniqueId"
        },
        "label": {
          "title": "Label",
          "type": "string"
        }
      },
      "required": ["label"]
    }
  },

  "items": {
    "$ref": "#/definitions/item"
  }
}
```

**Comment by germanbisurgi** [comment]
I checked and it works. But can you please post a direct link with the schema that is creating the issue?

**Comment by partikule** [comment]
Did you checked with the initUniqueId() callback ?
Try the method and the schema I provided, you will see the problem.

**Comment by schmunk42** [comment]
fixed in #790
